webapp/components/survey/CategoryDetails/store/useLocalState.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from 'react'
+
+import { useActions } from './actions'
+
+export const useLocalState = (props) => {
+  const { onCategoryCreated, categoryUuid } = props
+
+  const [state, setState] = useState({})
+
+  const Actions = useActions({ setState })
+
+  useEffect(() => {
+    Actions.init({ onCategoryCreated, categoryUuid })
```

Comment:
since you pass it here, you would avoid another redundant useParams

Reply:
ok